owlbot.py

```diff
@@ -142,12 +128,9 @@ def system(session, disable_grpc):
 # tests.
 num = s.replace(
     "CONTRIBUTING.rst",
-    """\
-\*\*\*\*\*\*\*\*\*\*\*\*\*
-Test Coverage
-\*\*\*\*\*\*\*\*\*\*\*\*\*
-""",
-    """\
+    'app_credentials.json"',
+    """app_credentials.json"
+
```

Comment:
This change is no longer correct after applying the generation of `CONTRIBUTING.rst` above:  see PR #166.